include/ssl-client/net/robocup_ssl_client/robocup_ssl_client.h

```diff
@@ -20,35 +20,38 @@
 //========================================================================
 #ifndef ROBOCUP_SSL_CLIENT_H
 #define ROBOCUP_SSL_CLIENT_H
-#include "net/netraw.h"
+
 #include <string>
-#include "pb/messages_robocup_ssl_detection.pb.h"
-#include "pb/messages_robocup_ssl_geometry.pb.h"
-#include "pb/messages_robocup_ssl_wrapper.pb.h"
-#include "pb/messages_robocup_ssl_refbox_log.pb.h"
-using namespace std;
+#include "ssl-client/net/netraw/netraw.h"
+#include "ssl-client/protobuf-files/protobuf-files.h"
```

Comment:
Como comentei acima, crei headers que inclui tudo que existe dentro de uma pasta. Assim se for de uma pasta vizinha, dá pra pegar todos os arquivos da outra dessa forma